Fix(cv_deploy): Fix workspace build error when deleting manifest configlets still assigned (#7001)
aristanetworks/avd: ansible_collections/arista/avd/roles/cv_deploy/README.md

!!! note "Root Containers Order"
    When initially deploying or adding new root containers, the role places its managed root containers to the top of the Studio container tree. Please be aware that this automated ordering **may displace any containers you have manually arranged**.

!!! warning "Manual configlet assignments"

>     Before you remove a configlet from the manifest, ensure it is not manually assigned to any non-manifest containers. If it is, you must manually remove it from those containers first.

Author: ```suggestion
    Before you remove a configlet created by a cv_deploy manifest, ensure it is not manually assigned to any non-manifest containers. Otherwise you must manually unassign the configlet from such containers first.
```

>     If a configlet managed by the manifest is manually assigned to a container that the manifest does **not** control, it must be manually unassigned before it is removed from the manifest.

Reviewer: Is this warning clear enough? I only understood what it means after reading actual code that handles it.
 - `configlet managed by the manifest` - is it a configlet which is a part of our input manifest? Or manifest-managed configlet in general (what was at one point pushed by the manifest but may not be part of the current manifest)
 - `container that the manifest does **not** control` - same thing, our current manifest or a `not manifest-namaged` container in general
 - `it must be manually unassigned before it is removed from the manifest` - `it` is container or configlet? `before it is removed from the manifest` - before we delete this configlet from CV or before we delete it from the input manifest (which doesn't make sense)?

Reviewer 2: A variant of Carl's proposal.
```suggestion
    Before you remove a configlet created by a cv_deploy manifest, ensure it is not manually assigned to any non-manifest containers. Otherwise you must manually unassign the configlet from such containers first.
```

Reviewer: Sounds very clear now!

>     If a configlet managed by the manifest is manually assigned to a container that the manifest does **not** control, it must be manually unassigned before it is removed from the manifest.

Author: ```suggestion
    Before you remove a configlet from the manifest, ensure it is not manually assigned to any non-manifest containers. If it is, you must manually remove it from those containers first.
```
